managed/ui/src/components/config/Security/certificates/AddCertificateForm.scss

```diff
@@ -0,0 +1,23 @@
+.suggestion {
+  position: relative;
+  top: -53px;
+  width: 100%;
+  height: 42px;
+  padding: 12px 13px;
+  font-size: 14px;
+  font-family: inherit;
+  z-index: 0;
+  color: #ccc;
+  background: transparent;
+  border: 1px solid #fff;
+  line-height: 1.42857143;
+}
+.search-container { 
+  height: 85px;
+}
+.search {
+  position: relative;
+  color: #000;
```

Comment:
It will be nice if use color variables - $YB_TEXT_COLOR instead static hashcode .

Reply:
For this one at other places, it is taken as the hash value.